sklearn/pipeline.py

```diff
@@ -69,6 +69,11 @@ class Pipeline(_BaseComposition):
     to another estimator, or a transformer removed by setting it to
     `'passthrough'` or `None`.
 
+    For examples use cases of `Pipeline` combined with
```

Comment:
Thank you, I have separates those. Though I left the second sentence below the first one instead of pulling it up, because it is indeed the more complex example.